.github/workflows/test-build-deploy.yml

```diff
@@ -0,0 +1,113 @@
+name: ci
+on:
+  push:
+    branches: [master]
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+** # Tag filters not as strict due to different regex system on Github Actions
+  pull_request:
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/cortexproject/build-image:update-golang-1.14.9-eb0c8d4d2
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v1
+      - name: Sym Link Expected Path to Workspace
```

Comment:
This linking is explained in README, apart from doing review, who reads that? I'd include the explanations from README directly into this file.

Reply:
Added a comment explaining this within the code itself in the subsequent PR to address this.
https://github.com/cortexproject/cortex/pull/3341